test/unit/full_text_search/attachment_test.rb

```diff
@@ -102,11 +102,7 @@ def teardown_server
     end
 
     def fixture_file_path(name)
-      dir = File.dirname(__FILE__)
-      path = Pathname(File.join(dir, "..", "..", "files", name)).expand_path
-      fixture_path = Pathname(self.class.fixture_path)
-      fixture_path += "files" if Rails::VERSION::MAJOR >= 6
-      path.relative_path_from(fixture_path)
+      Pathname(File.join(File.dirname(__FILE__), "..", "..", "files", name)).expand_path
```

Comment:
Can we simplify this?

```suggestion
      File.expand_path(File.join(File.dirname(__FILE__), "..", "..", "files", name))
```